src/middlewared/middlewared/plugins/account.py

```diff
@@ -88,6 +88,33 @@ class Config:
                 pass
         return user
 
+    @accepts(Ref('query-filters'), Ref('query-options'), Bool('dssearch', default=False))
+    async def query(self, filters=None, options=None, dssearch=False):
+        if not filters:
+            filters = []
+        filters += self._config.datastore_filters or []
+
+        options = options or {}
+        options['extend'] = self._config.datastore_extend
+        options['extend_context'] = self._config.datastore_extend_context
+        options['prefix'] = self._config.datastore_prefix
+
+        datastore_options = options.copy()
+        datastore_options.pop('count', None)
+        datastore_options.pop('get', None)
+
+        if dssearch:
```

Comment:
The above being said, I am not entirely sure how this block can be accomplished.

Perhaps we can look at the filters for `local = False` or `local != None` ?